compiler/x/codegen/OMRX86Instruction.cpp

```diff
@@ -434,14 +431,6 @@ TR::X86FenceInstruction::X86FenceInstruction(TR::Instruction   *precedingInstruc
    {
    }
 
-////////////////////////////////////////////////////////////////////////////////
-// TR::X86RestoreVMThreadInstruction:: member functions
-////////////////////////////////////////////////////////////////////////////////
-
-TR::X86RestoreVMThreadInstruction::X86RestoreVMThreadInstruction(TR_X86OpCodes op, TR::Node *node, TR::CodeGenerator *cg)
-   : TR::Instruction(node, op, cg)
-   {
-   }
 
```

Comment:
> I'm not sure what spaces you're asking to be cleaned up. Can you be more specific?

The double blank lines introduced by removal of these constructors. See line 433 and 434 after the change. There are several of these occurrences in this PR.